pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -231,8 +232,12 @@ func (pl *CSILimits) checkAttachableInlineVolume(vol v1.Volume, csiNode *storage
 		return fmt.Errorf("looking up provisioner name for volume %v: %w", vol, err)
 	}
 	if !isCSIMigrationOn(csiNode, inTreeProvisionerName) {
+		csiNodeName := ""
+		if csiNode != nil {
```

Comment:
The mock is working as expected even in the master code, even the test case said that migration is enabled, on line old 291, new 296:

```
if !isCSIMigrationOn(csiNode, pluginName) {
  klog.V(5).InfoS("CSI Migration of plugin is not enabled", "plugin", pluginName)
  return "", ""
}
```

This got returned.